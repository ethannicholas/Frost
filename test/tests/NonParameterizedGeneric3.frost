class NonParameterizedGeneric3 : Array {
}