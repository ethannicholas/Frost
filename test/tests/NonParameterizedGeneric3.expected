NonParameterizedGeneric3.panda:1:1: error: 'panda.collections.Array' expected 1 generic parameter, but found 0
1 error
